packages/core/rollup.config.js

```diff
@@ -15,7 +15,7 @@ const dependencies = Object.keys(pkg.dependencies)
       !['@rest-hooks/use-enhanced-reducer', '@babel/runtime'].includes(dep),
   );
 
-const extensions = ['.js', '.ts', '.tsx', '.mjs', '.json', '.node'];
+const extensions = ['.js', '.ts', '.tsx', '.mjs', '.json', '.node', 'jsx'];
```

Comment:
why does jsx omit the dot and the others include it?

Reply:
ack, good catch